package/MDAnalysis/topology/ParmEdParser.py

```diff
@@ -21,7 +21,6 @@
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
 import warnings
-from ..converters.ParmEdParser import ParmEdParser, squash_identical
```

Comment:
This is an intentional import, the class moved but we're keeping the import here for backwards compatibility

Reply:
To be clear to @vachram97 : do not remove the import